python/ray/experimental/workflow/BUILD

```diff
@@ -14,29 +14,7 @@ SRCS = [] + select({
 })
 
 py_test_module_list(
-  files = [
-      "tests/test_object_deref.py",
-      "tests/test_storage.py",
-      "tests/test_variable_mutable.py",
-      "tests/test_large_intermediate.py",
-      "tests/test_signature_check.py",
-  ],
-  size = "small",
-  extra_srcs = SRCS,
-  tags = ["exclusive"],
-  deps = ["//:ray_lib"],
-)
-
-py_test_module_list(
-  files = [
-      "tests/test_basic_workflows.py",
-      "tests/test_basic_workflows_2.py",
-      "tests/test_recovery.py",
-      "tests/test_lifetime.py",
-      "tests/test_workflow_manager.py",
-      "tests/test_virtual_actor.py",
-      "tests/test_storage_failure.py",
-  ],
+  files = glob(["tests/test_*.py", "examples/**/*.py"]),
```

Comment:
@iycheng I noticed some tests were missing from the list. OK with just using a wildcard here instead to define them?

Reply:
bazel query:
```
#workflow-examples/python/ray/experimental/workflow$ bazel query ...
DEBUG: /home/eric/Desktop/ray/bazel/ray_deps_setup.bzl:61:14: No implicit mirrors used because urls were explicitly provided
//python/ray/experimental/workflow:tests/test_workflow_manager
//python/ray/experimental/workflow:tests/test_virtual_actor_3
//python/ray/experimental/workflow:tests/test_virtual_actor_2
//python/ray/experimental/workflow:tests/test_virtual_actor
//python/ray/experimental/workflow:tests/test_variable_mutable
//python/ray/experimental/workflow:tests/test_storage_failure
//python/ray/experimental/workflow:tests/test_storage
//python/ray/experimental/workflow:tests/test_signature_check
//python/ray/experimental/workflow:tests/test_recovery
//python/ray/experimental/workflow:tests/test_object_deref
//python/ray/experimental/workflow:tests/test_lifetime
//python/ray/experimental/workflow:tests/test_large_intermediate
//python/ray/experimental/workflow:tests/test_dynamic_workflow_ref
//python/ray/experimental/workflow:tests/test_basic_workflows_2
//python/ray/experimental/workflow:tests/test_basic_workflows
//python/ray/experimental/workflow:examples/function_chain
//python/ray/experimental/workflow:examples/comparisons/metaflow/foreach_workflow
//python/ray/experimental/workflow:examples/comparisons/metaflow/foreach_metaflow
//python/ray/experimental/workflow:examples/comparisons/google_cloud_workflows/sub_workflows_workflow
//python/ray/experimental/workflow:examples/comparisons/google_cloud_workflows/data_cond_workflow
//python/ray/experimental/workflow:examples/comparisons/google_cloud_workflows/concat_array_workflow
//python/ray/experimental/workflow:examples/comparisons/cadence/trip_booking_workflow
//python/ray/experimental/workflow:examples/comparisons/cadence/sub_workflow_workflow
//python/ray/experimental/workflow:examples/comparisons/cadence/file_processing_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/retry_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/recursion_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/multi_step_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/loops_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/hello_world_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/exit_handler_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/dag_workflow
//python/ray/experimental/workflow:examples/comparisons/argo/conditionals_workflow
```

Reply:
Right now it's OK I think. But later when we have more tests, and eventually we'd like to split the tests if they require time more than LARGE.